go-packages/meep-net-char-mgr/algo-segment.go

```diff
@@ -208,14 +208,22 @@ func (algo *SegmentAlgorithm) ProcessScenario(model *mod.Model) error {
 			element.ZoneName = nodeCtx.Parents[mod.Zone]
 		}
 
+		deployment := model.GetNodeParent(element.DomainName).(*dataModel.Deployment)
+
 		// Set max App Net chars (use default if set to 0)
-		element.ConfiguredNetChar.Throughput = float64(proc.AppThroughput)
-		if element.ConfiguredNetChar.Throughput == 0 {
-			element.ConfiguredNetChar.Throughput = DEFAULT_THROUGHPUT_LINK
+		element.ConfiguredNetChar.Latency = float64(proc.NetChar.Latency)
+		element.ConfiguredNetChar.Jitter = float64(proc.NetChar.LatencyVariation)
+		element.ConfiguredNetChar.Distribution = deployment.NetChar.LatencyDistribution //set global value
+		element.ConfiguredNetChar.ThroughputDl = float64(proc.NetChar.ThroughputUl)
+		element.ConfiguredNetChar.ThroughputUl = float64(proc.NetChar.ThroughputUl)
+		element.ConfiguredNetChar.PacketLoss = float64(proc.NetChar.PacketLoss)
+
+		if element.ConfiguredNetChar.ThroughputUl == 0 {
+			element.ConfiguredNetChar.ThroughputUl = DEFAULT_THROUGHPUT_LINK
```

Comment:
I'm not sure why Throughput=0 case is not supported and reverts to default.
In an upcoming feature, when we get to simulate a disconnected UE, throughput = 0 will need to be supported

Reply:
At some point, we said a throughput of 0 is no bandwidth limitation, and since the default is at 1GBps, that's why it was set to that value. we said at this point that we could use 100% packet loss too, so that is why it was like that. Either way, it's only on ingress side that the traffic will be cut, either because of bw 0 or packet loss of 100. I don't think one is better than the other, TBD.